### ISSUE TITLE
Janggi color

### ISSUE THREAD

**Issue by CouchTomato87** [gbtami/pychess-variants#197]
Janggi color is a little confusing.

For starters, it may help to name the colors "Red" and "Blue." The current implementation has blue as white and red as black. This is a little confusing because I always treat red as "white" and blue as "black." In fact, red does have the first "move" by choosing setup.

Is it possible to change the player name dots to blue and red? This still doesn't fix the analysis bar and stuff.

I think what would be most consistent though is to make red correspond to white and blue/green correspond to black, keeping in line with all other variants. This would cause changes to the coding though I think, and also have similar black-white reversals as shogi.

**Comment by Little-Magician** [comment]
I don't think you have to worry about reversals like in shogi wouldn't the code change just require to change the pieces assigned to white and black.

**Comment by gbtami** [comment]
As I see the only thing we have to change are the little black and white dots in the game info block (upper left). After the pieces are set up, Cho (blue side) moves first, so everything else is Ok as it is now.

**Comment by CouchTomato87** [comment]
Well the corresponding colors for analysis (the bar and graph) would also have to match.

What's also confusing is the FEN. So w would be blue or red?

**Comment by gbtami** [comment]
I can agree that colorizing dots, gauge and grahp the same way would be nice.
At this moment only Shogi dots are adjusted.

In FEN ''w' is used to indicate the player who owns the upper case lettered pieces (and moves first in default starting position also). In Shogi it is called Black in Sittuyin Red in Janggi Blue in Xiangqi Red in western variants White.

**Comment by gbtami** [comment]
![http://i.imgur.com/UphPsji.png](http://i.imgur.com/UphPsji.png)
